### PR TITLE
FAI-817 - let ITs use trustyai-explainability's explainability-core

### PR DESCRIPTION
**Pull request by tteofili**
Stop using `explainability-core` from `kogito` and use `trustyai-explainability:explainability-core`.
See https://issues.redhat.com/browse/FAI-817

### CODE REVIEW

File: explainability-integrationtests/explainability-integrationtests-dmn/pom.xml [hunk 1]
```diff
@@ -2,19 +2,20 @@
...
-  <groupId>org.kie.trustyai</groupId>
   <artifactId>explainability-integrationtests-dmn</artifactId>
   <name>TrustyAI Explainability Integration Tests DMN</name>
 
   <dependencies>
     <dependency>
-      <groupId>org.kie.kogito</groupId>
+      <groupId>org.kie.trustyai</groupId>
       <artifactId>explainability-core</artifactId>
+      <version>${project.version}</version>
```
Comment (danielezonca) [inline explainability-integrationtests/explainability-integrationtests-dmn/pom.xml:18]: We need to define a BOM/parent with dependency management. Can you create a ticket?

File: explainability-integrationtests/explainability-integrationtests-opennlp/pom.xml [hunk 2]
```diff
@@ -2,19 +2,20 @@
...
-  <groupId>org.kie.trustyai</groupId>
   <artifactId>explainability-integrationtests-opennlp</artifactId>
   <name>TrustyAI Explainability Integration Tests OpenNLP</name>
 
   <dependencies>
     <dependency>
-      <groupId>org.kie.kogito</groupId>
+      <groupId>org.kie.trustyai</groupId>
       <artifactId>explainability-core</artifactId>
+      <version>${project.version}</version>
```
Comment (danielezonca) [inline explainability-integrationtests/explainability-integrationtests-opennlp/pom.xml:18]: Same here

File: explainability-integrationtests/explainability-integrationtests-pmml/pom.xml [hunk 3]
```diff
@@ -26,8 +26,9 @@
   <dependencies>
 
     <dependency>
-      <groupId>org.kie.kogito</groupId>
+      <groupId>org.kie.trustyai</groupId>
       <artifactId>explainability-core</artifactId>
+      <version>${project.version}</version>
```
Comment (danielezonca) [inline explainability-integrationtests/explainability-integrationtests-pmml/pom.xml:31]: And here

**Review by danielezonca — APPROVED** [review]

File: explainability-integrationtests/explainability-integrationtests-dmn/pom.xml (hunk 1 above)
Comment (tteofili) [reply explainability-integrationtests/explainability-integrationtests-dmn/pom.xml:18]: here it is https://issues.redhat.com/browse/FAI-819

**Review by RobGeada — APPROVED** [review]
LGTM!

**Pull request merged**